cmd/pressure-vessel/main.ino

```diff
@@ -0,0 +1,242 @@
+/*
+AUTHORS
+  Saxon Nelson-Milton <saxon@ausocean.org>
+
+LICENSE
+  Copyright (C) 2020 the Australian Ocean Lab (AusOcean)
+
+  It is free software: you can redistribute it and/or modify them
+  under the terms of the GNU General Public License as published by the
+  Free Software Foundation, either version 3 of the License, or (at your
+  option) any later version.
+
+  It is distributed in the hope that it will be useful, but WITHOUT
+  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+  for more details.
+
+  You should have received a copy of the GNU General Public License
+  in gpl.txt.  If not, see http://www.gnu.org/licenses.
+*/
+
+// Display pins.
+#define MAX7219DIN 3
+#define MAX7219CS 4
+#define MAX7219CLK 5
+
+// Display parameters.
+#define DISPLAY_BRIGHTNESS 15
+
+// Pin numbers.
+#define PRESSURE_PIN 0
+#define RELAY_PIN 2
+#define LED_PIN 13
+
+// Pressures.
+#define MAX_PRESSURE 150 //kPa
+#define RANGE 30 // kPa
+#define MIN_PRESSURE MAX_PRESSURE - RANGE
+#define ABS_MAX_PRESSURE 300 // kPa
+
+// Max pump running time.
+#define MAX_PUMP_TIME 0.5 // Minutes
+
+// Misc consts.
+#define NO_OF_READINGS 500
+
+// This will put arduino into an alarm state i.e. something
+// went wrong.
+void alarmed(){
```

Comment:
I'm thinking this should be a temporary state. Perhaps it can last as long as it takes for the pressure vessel to go back down to 0 kPa.